packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -123,6 +123,10 @@ class ImageCache {
   int get currentSize => _cache.length;
 
   /// Maximum size of entries to store in the cache in bytes.
+  /// This cache stores uncompressed images, so the size of each entry
+  /// in bytes is equal to the number of pixels in the image, which is often
+  /// much higher than its size before decoded by the codec. This value should
```

Comment:
nit: decompressed may be more idiomatic than uncompressed.

The size in bytes is not equal to the number of pixels, but maybe we could just say something like "the size of each entry is based on the number of pixels in the image".  I'd like to avoid being too precise here because the exact formula for determining this is an implementation detail that may change as we're able to make it more accurate.